buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkColors.kt

```diff
@@ -48,7 +55,7 @@ object BpkColor {
 
     override fun invoke(source: Map<String, Any>): BpkColors =
       parseColors(source, resolveReferences = false) {
-        !it.isSemanticColor() && !it.isMarcomms()
+        !it.isSemanticColor() && !it.hasSemanticSuffix(newSemanticSuffixes) && !it.isMarcomms()
```

Comment:
this is now only used in xml - we need to make sure we're filtering the right colours as it's slightly different to how the compose version was